packages/extension/src/components/pages/CreatePassword/CreatePassword.tsx

```diff
@@ -36,16 +42,25 @@ export const CreatePassword: FC<CreatePasswordProps> = ({
       setPasswordError('Passwords must match');
     } else {
       try {
-        saveWallet(wallet, confirmPasswordValue);
-        setActiveStep((prevActiveStep) => prevActiveStep + 1);
```

Comment:
this is removed?

Reply:
Yes, that was for testing purposes, thanks for reverting it :)